content/en/docs/ops/ambient/usage/waypoint/index.md

```diff
@@ -5,5 +5,261 @@ weight: 2
 owner: istio/wg-networking-maintainers
 test: no
 ---
--
-- This page is under construction.
+
+{{< warning >}}
+Ambient is currently in [alpha status](/docs/releases/feature-stages/#feature-phase-definitions).
+
+Please **do not run ambient in production** and be sure to thoroughly review the [feature phase definitions](/docs/releases/feature-stages/#feature-phase-definitions) before use.
+In particular, there are known performance, stability, and security issues in the `alpha` release. There are also functional caveats some of which are listed in the [Caveats section](#caveats) of this guide. There are also planned breaking changes, including some that will prevent upgrades. These are all limitations that will be addressed before graduation to `beta`. The current version of this guide is meant to assist early deployments and testing of the alpha version of `ambient`. The guide will continue to get updated as `ambient` itself evolves from alpha to beta status and beyond. 
+{{< /warning >}}
+
+## Introduction
+
+This guide provides instructions on how to set up and use the Waypoint proxy layer in Istio Ambient Mesh.
+
+Istio Ambient Mesh is a new way to deploy and manage microservices. In Ambient Mesh, workloads are no longer required to run sidecar proxies to participate in the service mesh. Ambient splits Istio’s functionality into two distinct layers, a secure overlay layer and a Layer 7 processing layer.
+
+Ztunnel proxy is used to handle L3 and L4 networking functions, such as mTLS authentication and L4 authorization. For workloads that require L7 networking features, such as HTTP load balancing and fault injection, a waypoint proxy can be deployed. The waypoint proxy is an optional component that is Envoy-based and is responsible for terminating workload HTTP traffic and parsing workload HTTP headers. They also enforce L7 policies and collect L7 metrics.
```

Comment:
```suggestion
Ztunnel is used to handle L3 and L4 networking functions, such as mTLS authentication and L4 authorization. The waypoint proxy is an optional component that is Envoy-based and is responsible for terminating workload HTTP traffic and parsing workload HTTP headers. For workloads that require L7 networking features, such as HTTP load balancing and fault injection, a waypoint proxy must be deployed.  It can also enforce L7 policies and collect L7 metrics.
```